games/h.yaml

```
@@ -526,13 +526,13 @@
   licenses:
   - As-is
   content: free
-  development: halted
+  development: active
```

```suggestion
  development: sporadic
```
last commit activity seems to be two months ago; based on OSGC definitions it should be sporadic